cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -161,6 +161,9 @@ controller, and serviceaccounts controller.`,
 // ResyncPeriod returns a function which generates a duration each time it is
 // invoked; this is so that multiple controllers don't get into lock-step and all
 // hammer the apiserver with list requests simultaneously.
+// This returns a function rather than just a duration because it is stored
+// on the ControllerContext to be invoked multiple times.
+// If you just need to use it once, immediately invoke it.
```

Comment:
the added comment seems to repeat the original comment with little extra info. consider merging it into the original comment.

Reply:
done